sechub-pds-solutions/iac/tests/clone_repo.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: MIT
+
+current_test_folder="$1"
+vulnerable_repo="$2"
+	
+if [[ ! -d "$current_test_folder" ]]
+then
+	echo "Target folder is empty"
+	exit 1
+fi
+
+if [[ -z "$vulnerable_repo" ]]
+then
+	echo "No vulnerable application repository provided"
+	exit 1
```

Comment:
You could use a different exit code because it's a different reason:
```suggestion
	exit 2
```